tests/helpers/with-page-auth-required.test.ts

```diff
@@ -16,8 +16,9 @@ describe('with-page-auth-required ssr', () => {
   });
 
   test('allow access to a page with a valid session', async () => {
-    const baseUrl = await setup(withoutApi);
+    const baseUrl = await setup(withoutApi, { withPageAuthRequiredOptions: { returnTo: '/foo' } });
```

Comment:
Thanks for the review. I reverted it back and added your suggestions. Also improved jest config so it will look for tests only in `tests` directory. Before it was also grabbing `cypress` tests files. Probably because of matching test file pattern :/